### PR TITLE
Next.js 9.2 Blog Post

### PR DESCRIPTION
**Pull request by Timer**
Adds the blogpost for 9.2.

### CODE REVIEW

**Comment by zeit-team** [comment]
#### 📝Changed routes:


| [**/blog/next-9-2**](https://next-site-git-blog-92.zeit.now.sh/blog/next-9-2) |
|:-:|
|<a href="https://next-site-git-blog-92.zeit.now.sh/blog/next-9-2">Screenshot of /blog/next-9-2</a><br /><sup><a href="https://images.weserv.nl?url=https://microlink.nyc3.digitaloceanspaces.com/DC2yDIs6lJUb3PtewJCFQLyluns_">(view full size)</a> |







Commit 2b6051fb737a76f4451d54e7fe1c0f00621b68ec (https://next-site-git-blog-92.zeit.now.sh).

File: pages/blog/next-9-2.mdx [hunk 1]
```diff
@@ -0,0 +1,479 @@
...
+
+Because all of these bundles are required for the application to become interactive, it's important they are as optimized as possible.
+In practice, this means not over-downloading code from other parts of the application.
+
+For this reason, Next.js used a `commons` bundle that held common JavaScript between pages.
+The calculation of the old bundle-splitting strategy to generate `commons` was a usage-ratio heuristic.
+If a module was used in more than 50% of all pages, it'd be marked as a common module.
+Otherwise, it was bundled in the page's JavaScript file.
+
+However, applications could exist of many different types of pages.
```
Comment (housseindjirdeh) [inline pages/blog/next-9-2.mdx]: Feel free to ignore, but I assume this was meant to say:

```suggestion
However, applications can consist of many different types of pages.
```

Comment (Timer) [reply pages/blog/next-9-2.mdx]: Thanks!

File: pages/blog/next-9-2.mdx [hunk 2]
```diff
@@ -0,0 +1,479 @@
...
+In practice, this means not over-downloading code from other parts of the application.
+
+For this reason, Next.js used a `commons` bundle that held common JavaScript between pages.
+The calculation of the old bundle-splitting strategy to generate `commons` was a usage-ratio heuristic.
+If a module was used in more than 50% of all pages, it'd be marked as a common module.
+Otherwise, it was bundled in the page's JavaScript file.
+
+However, applications can consist of many different types of pages.
+For example, marketing pages, a blog, and a dashboard.
+If there was a large number of marketing pages compared to other page types, the commons calculation would cause a more optimized result for the marketing pages.
```
Comment (msweeneydev) [inline pages/blog/next-9-2.mdx]:
```suggestion
If there was a large number of marketing pages compared to other page types, the commons calculation would result in optimizations heavily focused on the marketing pages.
```

File: pages/blog/next-9-2.mdx [hunk 3]
```diff
@@ -0,0 +1,479 @@
...
+
+Our goal is to optimize for all page types, in one application.
+
+[Alex Castle](https://twitter.com/atcastle) [proposed a new method of chunking](https://github.com/zeit/next.js/issues/7631) (creation of separate JavaScript files) that allows more optimized commons chunking with multiple files, especially when many pages are involved.
```
Comment (msweeneydev) [inline pages/blog/next-9-2.mdx]:
```suggestion
[Alex Castle](https://twitter.com/atcastle) [proposed a new method of chunking](https://github.com/zeit/next.js/issues/7631) (creation of separate JavaScript files) that allows optimized commons chunking with multiple files, including when many page types are involved.
```

File: pages/blog/next-9-2.mdx [hunk 4]
```diff
@@ -0,0 +1,479 @@
...
+
+Today, we're happy to announce this new chunking behavior is on-by-default in Next.js 9.2.
```
Comment (msweeneydev) [inline pages/blog/next-9-2.mdx]:
```suggestion
Today, we're happy to announce this new chunking behavior is enabled by default in Next.js 9.2.
```

File: pages/blog/next-9-2.mdx [hunk 5]
```diff
@@ -0,0 +1,479 @@
...
+We'd like to extend deep gratitude to the [Google Chrome team](/blog/next-9#google-chrome-collaboration) and [Alex Castle](https://twitter.com/atcastle) for contributing this change.
+This change reflects the cumulative effort of weeks of research, lab testing, real-world testing, and implementation.
+
+The new chunking implementation leverages [HTTP/2](https://developers.google.com/web/fundamentals/performance/http2) to deliver more, smaller chunks.
```
Comment (msweeneydev) [inline pages/blog/next-9-2.mdx]:
```suggestion
The new chunking implementation leverages [HTTP/2](https://developers.google.com/web/fundamentals/performance/http2) to deliver a greater number of smaller sized chunks.
```

File: pages/blog/next-9-2.mdx [hunk 6]
```diff
@@ -0,0 +1,479 @@
...
+
+Under the new heuristic, chunks are created for:
+
+- A minimal chunk for each page
```
Comment (msweeneydev) [inline pages/blog/next-9-2.mdx]:
```suggestion
- A minimal chunk for each page.
```

File: pages/blog/next-9-2.mdx [hunk 7]
```diff
@@ -0,0 +1,479 @@
...
+- A framework chunk containing React, ReactDOM, React's Scheduler, etc
```
Comment (msweeneydev) [inline pages/blog/next-9-2.mdx]:
```suggestion
- A framework chunk containing React, ReactDOM, React's Scheduler, etc.
```

File: pages/blog/next-9-2.mdx [hunk 8]
```diff
@@ -0,0 +1,479 @@
...
+- Library chunks for any `node_module` dependency over 160kb (pre-minify/gzip)
+- A commons chunk for code used across all pages
```
Comment (msweeneydev) [inline pages/blog/next-9-2.mdx]:
```suggestion
- A commons chunk for code used across all pages.
```

File: pages/blog/next-9-2.mdx [hunk 9]
```diff
@@ -0,0 +1,479 @@
...
+- As many shared chunks (used by 2 or more pages) as possible, optimizing for overall application size and initial load speed
```
Comment (msweeneydev) [inline pages/blog/next-9-2.mdx]:
```suggestion
- As many shared chunks (used by 2 or more pages) as possible, optimizing for overall application size and initial load speed.
```

File: pages/blog/next-9-2.mdx [hunk 10]
```diff
@@ -0,0 +1,479 @@
...
+- Next.js' client-side runtime
```
Comment (msweeneydev) [inline pages/blog/next-9-2.mdx]:
```suggestion
- Next.js' client-side runtime.
```

File: pages/blog/next-9-2.mdx [hunk 11]
```diff
@@ -0,0 +1,479 @@
...
+- Webpack runtime
```
Comment (msweeneydev) [inline pages/blog/next-9-2.mdx]:
```suggestion
- Webpack runtime.
```

File: pages/blog/next-9-2.mdx [hunk 12]
```diff
@@ -0,0 +1,479 @@
...
+        <td>
+          <b>70%</b>
+          <em>smaller</em>
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+The new chunking behavior does not only reduce your overall and initial-load size, but also successive client-side navigations.
```
Comment (msweeneydev) [inline pages/blog/next-9-2.mdx]:
```suggestion
The new chunking behavior not only reduces your overall and initial-load size, but also successive client-side navigations.
```

**Pull request merged**